.eslintrc.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "amex"
+  "extends": "amex",
+  "ignorePatterns": ["dist/**/*"]
```

Comment:
even though it is not merged yet, we are going for a much more strict eslint config;

https://github.com/americanexpress/vitruvius/pull/57/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090f

this config applies strict-type-checked rules to all source code: https://github.com/americanexpress/vitruvius/pull/57/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR32

but allows for less strict unit test writing : https://github.com/americanexpress/vitruvius/pull/57/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR13


Please adopt this eslint config, and then make sure all its rules are followed